corehq/apps/linked_domain/tests/test_linked_apps.py

```diff
@@ -551,3 +552,51 @@ def _get_image_data():
     image_path = os.path.join('corehq', 'apps', 'hqwebapp', 'static', 'hqwebapp', 'images', 'commcare-hq-logo.png')
     with open(image_path, 'rb') as f:
         return f.read()
+
+
+class TestLinkedAppsWithShadowForms(TestCase):
+    maxDiff = None
```

Comment:
What's this?

Reply:
output control - otherwise unittest will truncate the error diff if it's longer than some number of characters.